packages/ui/src/ui/store/actions/global/index.ts

```diff
@@ -313,9 +313,12 @@ export function mergeScreen() {
     };
 }
 
-export function handleAuthError() {
+export function handleAuthError({ytAuthCluster}: {ytAuthCluster?: string} = {}) {
     if (getConfigData().allowLoginDialog) {
-        getWindowStore().dispatch({type: GLOBAL_PARTIAL, data: {showLoginDialog: true}});
+        getWindowStore().dispatch({
```

Comment:
Probably it is better to display some toaster-error if ytAuthCluster is not defined